frontend/src/components/reports/study/common/ReportByDateCSV.js

```diff
@@ -0,0 +1,293 @@
+      return {
+        startDate: null,
+        endDate: null,
+        error: null,
+        studyType: null,
+        dateType: null,
+      };
+    } else {
+      return {
+        startDate: null,
+        endDate: null,
+        error: null,
+        studyType: null,
+      };
+    }
+  });
+
+  const handleDatePickerChangeDate = (datePicker, date) => {
+    let updatedDate = encodeDate(date);
+    let obj = null;
+    switch (datePicker) {
+      case "startDate":
+        obj = {
+          ...reportFormValues,
+          startDate: updatedDate,
+        };
+        break;
+      case "endDate":
+        obj = {
+          ...reportFormValues,
+          endDate: updatedDate,
+        };
+        break;
+      default:
+    }
+    setReportFormValues(obj);
+  };
+
+  const handleSubmit = () => {
+    if (!reportFormValues.startDate || !reportFormValues.endDate) {
+      setReportFormValues({
+        ...reportFormValues,
+        error: intl.formatMessage({
+          id: "error.dateRange.start",
+          defaultMessage: "Please select Start and end date.",
+        }),
+      });
+      return;
+    }
+
+    if (!reportFormValues.studyType) {
+      setReportFormValues({
+        ...reportFormValues,
+        error: intl.formatMessage({
+          id: "error.report.csv.study",
+          defaultMessage: "Please select study type.",
+        }),
+      });
+      return;
+    }
+    if (props.report === "CIStudyExport" && !reportFormValues.dateType) {
+      setReportFormValues({
+        ...reportFormValues,
+        error: intl.formatMessage({
+          id: "error.report.csv.dateType",
+          defaultMessage: "Please select date type.",
+        }),
+      });
+      return;
+    }
+
+    setReportFormValues({
+      ...reportFormValues,
+      error: "",
+    });
+
+    setLoading(true);
+
+    const baseParams = `report=${props.report}&type=patient`;
+    const baseUrl = `${config.serverBaseUrl}/ReportPrint`;
+    const additionalParams =
+      props.report === "CIStudyExport"
+        ? `projectCode=${reportFormValues.studyType}&dateType=${reportFormValues.dateType}`
+        : `vlStudyType=${reportFormValues.studyType}`;
+    const url = `${baseUrl}?${baseParams}&upperDateRange=${reportFormValues.endDate}&lowerDateRange=${reportFormValues.startDate}&${additionalParams}`;
+
+    window.open(url, "_blank");
+    setLoading(false);
+    setNotificationVisible(true);
+  };
+
+  const statusOptions =
+    props.report === "CIStudyExport"
+      ? [
+          {
+            id: "24",
+            value: "Initial ARV",
+          },
+          {
+            id: "27",
+            value: "Follow-up ARV",
+          },
+          {
+            id: "25",
+            value: "EID",
+          },
+          {
+            id: "28",
+            value: "Viral Load",
+          },
+          {
+            id: "21",
+            value: "Indeterminate",
+          },
+        ]
+      : [
+          {
```

Comment:
Same for thse Poject codes , this is the List of [Projects](https://github.com/mozzy11/OpenELIS-Global-2/blob/develop_3x/src/main/java/org/openelisglobal/reports/action/implementation/ExportTrendsByDate.java#L238-L263)